test/unit/callbacks.js

```diff
@@ -20,15 +20,15 @@ var output,
 	outputB = addToOutput( "B" ),
 	outputC = addToOutput( "C" ),
 	tests = {
-		"":                   "XABC   X     XABCABCC  X  XBB X   XABA  X   XX",
-		"once":               "XABC   X     X         X  X   X   XABA  X   XX",
-		"memory":             "XABC   XABC  XABCABCCC XA XBB XB  XABA  XC  XX",
-		"unique":             "XABC   X     XABCA     X  XBB X   XAB   X   X",
-		"stopOnFalse":        "XABC   X     XABCABCC  X  XBB X   XA    X   XX",
-		"once memory":        "XABC   XABC  X         XA X   XA  XABA  XC  XX",
-		"once unique":        "XABC   X     X         X  X   X   XAB   X   X",
-		"once stopOnFalse":   "XABC   X     X         X  X   X   XA    X   XX",
-		"memory unique":      "XABC   XA    XABCA     XA XBB XB  XAB   XC  X",
```

Comment:
This used to be ASCII-art and is now gibberish. Can we make an exception here?

Reply:
good point. I just need to figure out how to make that exception.

Reply:
got it. It's key-spacing.